tool/actions-gh-release/release.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 var (
-	releaseNoteBlockRegex = regexp.MustCompile(`(?s)(?:Release note\*\*:\s*(?:<!--[^<>]*-->\s*)?` + "```(?:release-note)?|```release-note)(.+?)```")
+	releaseNoteBlockRegex      = regexp.MustCompile(`(?s)(?:Release note\*\*:\s*(?:<!--[^<>]*-->\s*)?` + "```(?:release-note)?|```release-note)(.+?)```")
+	releaseNotePullNumberRegex = regexp.MustCompile(`#[0-9]+`)
```

Comment:
```suggestion
	releaseNotePullNumberRegex = regexp.MustCompile(`#\d{4}`)
```
nits

Reply:
I think this doesn't work because `#1` and `#12345` also may exist. wdyt?

Reply:
Ah, you're right. I missed it with the commit id 😄